glossary.yml

```
@@ -2764,6 +2990,12 @@
       element of the source vector, i.e., `result[index[i]] = source[i]`.  Push
       indexing can easily produce gaps and collisions.
 
+- slug: python
+  en:
+    term: "Python"
+    def: >
+      A popular open-source programming language.
```

This definition felt a bit empty, and I think this may be a good addendum (but needs interpreted and compiled to be defined)

```suggestion
      A popular interpreted open-source programming language that relies on white space indentation to define control structure.
```

---

Added.